backend/cms/fixtures/test_data.json

```diff
@@ -735,7 +735,7 @@
 and freedom of reporting by means of broadcasts and films shall be guaranteed. There shall be no censorship.\r\n\r\n(...)\r\n\r\nMarriage and the Family; Children Born Outside of Marriage\r\n\r\nArticle 6\r\n\r\n(1) Marriage and the family shall enjoy the special protection of the state.\r\n\r\n(...)\r\n\r\nHere you can find the Basic Law in 11 languages:* \u00bb\u00bbArabic, \u00bb\u00bbChinese, \u00bb\u00bbEnglish, \u00bb\u00bbFrench, \u00bb\u00bbItalian, \u00bb\u00bbPersian, \u00bb\u00bbPolish, \u00bb\u00bbRussian, \u00bb\u00bbSerbian, \u00bb\u00bbSpanish, \u00bb\u00bbTurkish and of course in \u00bb\u00bbGerman.\r\n\r\n*It was not possible to find this kind of linguistic diversity on any other website.\r\n\r\nHow the German legal state works is illustrated visually in this film: \u00bb\u00bbGerman, \u00bb\u00bbDari, \u00bb\u00bbArabic, \u00bb\u00bbEnglish, \u00bb\u00bbFrench, \u00bb\u00bbPashto, \u00bb\u00bbUrdu.\r\nUniversal Declaration of Human Rights\r\n\r\nHuman rights are important rights that apply to everyone all over the world. Human rights are rights that every person has because of their humanity. They protect the dignity of every human being and apply to every person equally, regardless of \"race, colour, sex, language, religion, political or other opinion, national or social origin, property, birth or other status.\" (Article 2. of the United Nations' Universal Declaration of Human Rights, issued on 10/12/1948)\r\n\r\nThere are 30 rights. The most important are:\r\n\r\n    All people are of equal value.\r\n    Everyone is allowed to have their own opinion.\r\n    Right to peace and safety.\r\n\r\nVisit the Road of Human Rights! Each of the elements in the Road of Human Rights takes one of the human rights articles in German and in another language. The Road of Human Rights is both a denunciation of the crimes committed by the Nazis and a reminder that human rights are still being severely violated in many countries worldwide.\r\n\r\nRoad of Human Rights (Stra\u00dfe der Menschenrechte)\r\n Kart\u00e4usergasse 1, 90402 N\u00fcrnberg\r\nChildren's Rights\r\n\r\nChildren need offer protection, which is why they also have special rights. The individual children's rights are stipulated in the Convention on the Rights of the Child. This was adopted by the UN General Assembly in 1989 and subsequently ratified by 193 countries. The UN Convention on the Rights of the Child includes 54 articles.\r\n\r\nIn Nuremberg City Park can be found the Road of Children\u2019s Rights. Here, children can learn more about their rights at eight different points. You can find more information here: \u00bb\u00bbActivities for Children and Teenagers.\r\n\r\nChildren Have Rights Too is the annual theme of the Alliance for Family in Nuremberg. More information on children's rights in Nuremberg: \u00bb\u00bbChildren's Rights in Nuremberg\r\n\r\nThe 10 most important Children\u2019s rights are visually explained here:\u00bb\u00bbArabic and \u00bb\u00bbPersian.",
```

Comment:
```suggestion
      "text": "The Basic Law\r\n\r\nIn every country, there are rules that everyone has to adhere to. The most important law in Germany is the Basic Law. It is a collection of 146 articles. Every article stands for a law \u2013 i.e., a rule.These rules govern community life in Germany.The fundamental rights protect the freedom of each individual. They are stipulated in Articles 1 to 19 of the Basic Law (Catalogue of Fundamental Rights).\r\n\r\nProtection of Human Dignity\r\n\r\nArticle 1\r\n\r\n(1) Human dignity shall be inviolable. To respect and protect it shall be the duty of all state authority.\r\n\r\n(...)\r\n\r\nPersonal Freedoms\r\n\r\nArticle 2\r\n\r\n(1) Every person shall have the right to free development of his personality insofar as s/he does not violate the rights of others or offend against the constitutional order or the moral law.\r\n\r\n(...)\r\n\r\nThis also means the right to sexual self-determination. This means: All human beings can decide for themselves which sexual acts they wants to participate in and nobody can be forced to do something they do not want to do. More information: \u00bb\u00bb Sexual Self-Determination\r\n\r\nEquality Before the Law\r\n\r\nArticle 3\r\n\r\n(1) All persons shall be equal before the law.\r\n\r\n(2) Men and women shall have equal rights. The state shall promote the actual implementation of equal rights for women and men and take steps to eliminate disadvantages that now exist.\r\n\r\n www.zanzu.de\r\n\r\n(3) No person shall be favoured or disfavoured because of sex, parentage, race, language, homeland and origin, faith, or religious or political opinions. No person shall be disfavoured because of disability.\r\n\r\nFreedom of Faith, Conscience and Creed\r\n\r\nArticle 4\r\n\r\n(1) Freedom of faith and of conscience, and freedom to profess a religious or philosophical creed, shall be inviolable.\r\n\r\n(...)\r\n\r\nFreedom of Expression\r\n\r\nArticle 5\r\n\r\n(1) Every person shall have the right freely to express and disseminate her/his opinions in speech, writing, and pictures and to inform herself/himself without hindrance from generally accessible sources. Freedom of the press and freedom of reporting by means of broadcasts and films shall be guaranteed. There shall be no censorship.\r\n\r\n(...)\r\n\r\nMarriage and the Family; Children Born Outside of Marriage\r\n\r\nArticle 6\r\n\r\n(1) Marriage and the family shall enjoy the special protection of the state.\r\n\r\n(...)\r\n\r\nHere you can find the Basic Law in 11 languages:* \u00bb\u00bbArabic, \u00bb\u00bbChinese, \u00bb\u00bbEnglish, \u00bb\u00bbFrench, \u00bb\u00bbItalian, \u00bb\u00bbPersian, \u00bb\u00bbPolish, \u00bb\u00bbRussian, \u00bb\u00bbSerbian, \u00bb\u00bbSpanish, \u00bb\u00bbTurkish and of course in \u00bb\u00bbGerman.\r\n\r\n*It was not possible to find this kind of linguistic diversity on any other website.\r\n\r\nHow the German legal state works is illustrated visually in this film: \u00bb\u00bbGerman, \u00bb\u00bbDari, \u00bb\u00bbArabic, \u00bb\u00bbEnglish, \u00bb\u00bbFrench, \u00bb\u00bbPashto, \u00bb\u00bbUrdu.\r\nUniversal Declaration of Human Rights\r\n\r\nHuman rights are important rights that apply to everyone all over the world. Human rights are rights that every person has because of their humanity. They protect the dignity of every human being and apply to every person equally, regardless of \"race, colour, sex, language, religion, political or other opinion, national or social origin, property, birth or other status.\" (Article 2. of the United Nations' Universal Declaration of Human Rights, issued on 10/12/1948)\r\n\r\nThere are 30 rights. The most important are:\r\n\r\n    All people are of equal value.\r\n    Everyone is allowed to have their own opinion.\r\n    Right to peace and safety.\r\n\r\nVisit the Road of Human Rights! Each of the elements in the Road of Human Rights takes one of the human rights articles in German and in another language. The Road of Human Rights is both a denunciation of the crimes committed by the Nazis and a reminder that human rights are still being severely violated in many countries worldwide.\r\n\r\nRoad of Human Rights (Stra\u00dfe der Menschenrechte)\r\n Kart\u00e4usergasse 1, 90402 N\u00fcrnberg\r\nChildren's Rights\r\n\r\nChildren need extra protection, which is why they also have special rights. The individual children's rights are stipulated in the Convention on the Rights of the Child. This was adopted by the UN General Assembly in 1989 and subsequently ratified by 193 countries. The UN Convention on the Rights of the Child includes 54 articles.\r\n\r\nIn Nuremberg City Park can be found the Road of Children\u2019s Rights. Here, children can learn more about their rights at eight different points. You can find more information here: \u00bb\u00bbActivities for Children and Teenagers.\r\n\r\nChildren Have Rights Too is the annual theme of the Alliance for Family in Nuremberg. More information on children's rights in Nuremberg: \u00bb\u00bbChildren's Rights in Nuremberg\r\n\r\nThe 10 most important Children\u2019s rights are visually explained here:\u00bb\u00bbArabic and \u00bb\u00bbPersian.",
```